tony-core/src/main/java/com/linkedin/tony/TonyConfigurationKeys.java

```diff
@@ -5,6 +5,20 @@
 package com.linkedin.tony;
 
 public class TonyConfigurationKeys {
+  public enum MLFramework {
+    TENSORFLOW("tensorflow"),
+    PYTORCH("pytorch");
+
+    private final String frameworkName;
```

Comment:
I think this String field is unnecessary. By default, if you use
```
  public enum MLFramework {
    TENSORFLOW,
    PYTORCH
  }
```

MLFramework.toString() will return TENSORFLOW or PYTORCH.